### PR TITLE
Ensure some .Dispose are called with using blocks

### PR DESCRIPTION
**Pull request by manuc66**
Hi,

By looking at the code I found that some Disposable Object are not disposed

This PR suggest to fix some of them. 

Thanks for the plugin !

### CODE REVIEW

File: KeePassRPC/DpiFix.cs [hunk 1]
```diff
@@ -32,9 +32,9 @@ public static Image ScaleImageTo16x16(Image img, bool bForceNewObject)
                 rSource.Offset(-0.5f, -0.5f);
 
                 g.DrawImage(img, rDest, rSource, GraphicsUnit.Pixel);
-            }
 
-            return bmp;
+                return bmp;
```
Comment (dlech) [inline KeePassRPC/DpiFix.cs:37]: How does the caller use the returned `bmp` after it has been disposed?

**Review by dlech — COMMENTED** [review]
Have you observed any actual problems caused by not disposing any of these objects? Are there any unmanaged resources you are worried about leaking if the finalizer is not called?

File: KeePassRPC/DpiFix.cs (hunk 1 above)
Comment (manuc66) [reply KeePassRPC/DpiFix.cs:37]: You're right it shouldn't be disposed there

**Comment by manuc66** [comment]
You can see that the memory usage of the kepass.exe process grows
- each time you display a login form
- when the KeePass RPC option dialog is open and closed (very little)

**Comment by luckyrat** [comment]
Thanks @manuc66 and thanks @dlech for the review. I'm going to merge this in for a release in the next week or so and also take a look to see if there are any other similar issues that might lead to resource leaks.

**Pull request merged**